.github/workflows/codeql.yml

```diff
@@ -29,6 +29,7 @@ jobs:
     # Consider using larger runners for possible analysis time improvements.
     runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
     timeout-minutes: ${{ (matrix.language == 'swift' && 120) || 360 }}
+    if: github.repository == 'gui-cs/Terminal.Gui'|| github.event_name == 'schedule'
```

Comment:
This is the best part of it all. Thanks for getting fed up and doing it! :)

It's so noisy for forks, for various other reasons.

Reply:
No worries, thank you for the feedback, very appreciated! :smile: